doc/api/path.md

```diff
@@ -235,11 +235,11 @@ On Windows:
 
 ```js
 path.format({
-    root : "C:\\",
     dir : "C:\\path\\dir",
+    root : "C:\\",
     base : "file.txt",
-    ext : ".txt",
-    name : "file"
+    name : "file",
```

Comment:
I'd agree not necessary, but these same five props are listed in five different places on the page, in different orders. The change just puts them all in the same order: `dir`, `root`, `base`, `name`, `ext`.